src/devtools/client/inspector/rules/components/RulesApp.tsx

```diff
@@ -9,9 +9,14 @@ import { Toolbar } from "devtools/client/inspector/rules/components/Toolbar";
 import { getStr } from "devtools/client/inspector/rules/utils/l10n";
 import { useSelector } from "react-redux";
 import { RulesState, RuleState } from "../state/rules";
+import { RuleInheritance } from "../models/rule";
+import _ from "lodash";
```

Comment:
You don't seem to be using lodash in this file but even if you did you shouldn't import it like this, otherwise all of lodash will be included in the bundle. We always import single functions from lodash, e.g. `import throttle from "lodash/throttle";`

Reply:
My keyboard is acting up and while doing auto-complete it accidentally does a selection and IDE then auto-imports what it can, but I should be more careful when doing a cleanup later. Good catch, thanks 👍